vaadin-grid-flow-parent/vaadin-grid-flow-integration-tests/frontend/benchmark.js

```diff
@@ -16,7 +16,7 @@ style.appendChild(
     }
   }
 
-  vaadin-grid-tree-toggle[leaf] {
+  [benchmark] vaadin-grid-tree-toggle[leaf] {
```

Comment:
`:host()` isn't used here because these styles end up in the global scope (`<vaadin-grid-tree-toggle>` is in the `<vaadin-grid>`'s light DOM)